### DISCUSSION TITLE
MSU and alternate Link sprites using the flatpak on Steam Deck

### DISCUSSION THREAD

**Discussion by Packers4Evar12** [snesrev/zelda3#265] [Q&A]
Where should I put the MSU and sprite files if I installed the zelda3 flatpak?

Also, if I add the game to Steam it first opens the dialog box to open the .ini file, which is fine but not ideal in game mode. Is there an AppImage or executable file that I can add to Steam instead?

**Comment by Arrow-Loop** [comment]
I'm having the same issue. I really don't understand the file layout in Linux, it doesn't make any sense to me. The minimal instructions for adding a sprite just say to place a sprites-gfx folder in the root of the zelda3 folder, but I can't tell which folder is the actual zelda3 folder. There are there are files all of the place in Linux for an app, some in a flatpak directory some in the .local directory some in an Applications directory... I'm so confused.

**Reply by Packers4Evar12** [reply]
I actually figured out where to put the sprites and MSU files. 
![show-hidden-files](https://github.com/snesrev/zelda3/assets/10317298/6a0f410d-0bd9-447d-8a0d-664a32844353)

home/deck/.var/app/io.github.snesrev.Zelda3/data
